storage_integration_test.go

```diff
@@ -147,7 +147,7 @@ func TestIntegration(t *testing.T) {
 		defer func() {
 			b := backoff.NewExponentialBackOff()
 			b.MaxElapsedTime = 0
-			backOff := backoff.WithMaxRetries(b, 7)
+			backOff := backoff.WithMaxTries(b, 7)
```

Comment:
The integration tests were broken for quite some time now. There was some vendor magic switching back and forth between master and a dedicated release. For external deps we should stick to releases if possible. Thus I reverted this here.